nemo/collections/tts/modules/tacotron2.py

```diff
@@ -113,12 +113,12 @@ def forward(self, *, token_embedding, token_len):
         return outputs
 
     def save_to(self, save_path: str):
-        # TODO: Implement me!!!
+        # TODO: Implement me!
```

Comment:
```python
class FileIO(ABC):
    @abstractmethod
    def save_to(self, save_path: str):
        """Saves module/model with weights"""
        pass

    @classmethod
    @abstractmethod
    def restore_from(cls, restore_path: str):
        """Restores module/model with weights"""
        pass
```
I cannot create a NeuralModule without defining these functions,

Reply:
I also don't know how I'm supposed to implement these functions

Reply:
Save to and restore from are already implemented in ModelPT. All subclasses have it predefined so you can just try loading a pretrained model (using explicit constructor and load weights), use save_to() to create a .Nemo file, and immediate restore with .restlre_from() and check if inference is correct c

Reply:
If it's not a subclass of ModelPT, then @okuchaiev is it possible to move the save to restore from impl into the actual FileIO class itself, so that sunclasses don't need to implement it ?

Reply:
And worst case if that you could try copy pasting the code from the ModelPT impl into this and see if it works (and since it's a class method, maybe just do a dispatched call to the MldelPT method itself without copy pasting ?

Reply:
We should think about save_to/restore_from for NeuralModules as opposed to Models. E.g. should we have it?
>>"you could try copy pasting the code from the ModelPT impl into this and see if it works"
- let's not do this. 

I would leave it as - is in this PR. NeuralModule's saving/restoring is something we should think about improving in 1.0.0